Gemfile

```diff
@@ -19,6 +19,7 @@ group :development do
   gem "pry"
   gem "rubocop"
   gem "test-unit"
+  gem "webrick"
```

Comment:
Is there any reason because this was missing?

Reply:
webrick is part of the standard library and comes with every (normal) ruby install so we don't include it. Although, there have been discussions to include things like webrick I'm not currently a fan but still pondering the issue.

Reply:
For now, I'd prefer if you removed it

Reply:
Got it. Any chance to comment this, maybe in the contributing guidelines?

Reply:
I'm not really sure - how did you even notice this? It's been like this for years and I've rarely seen anyone add stdlib gems to their Gemfiles... am I just... _old_ and that's what the cool kids do these days?

Reply:
@PragTob The thing is, `webrick` has been removed from stdlib in ruby 3.0. So people who depend on `webrick` and were being explicit about it will not even notice the change. However, if you depended on `webrick` but were relying on `ruby` to provide that dependency, then things will break for you.

Reply:
@deivid-rodriguez thanks, the problem is with Ruby 3.0 indeed

I think I need to edit the commit message and clarify with a comment why there is a (former) stdlib in the Gemfile

Reply:
@deivid-rodriguez thanks, didn't know! Guess gotta read up on my Ruby 3. In that case @tagliala please leave it in!

Reply:
The release notes don't mention it though :thinking:  https://www.ruby-lang.org/en/news/2020/12/20/ruby-3-0-0-rc1-released/

Mentions some other gems... huh.

Here is the issue removing it though: https://bugs.ruby-lang.org/issues/17303

Reply:
I've changed the commit message and left an explicit comment on why `webrick` is there